packages/orbit-components/src/primitives/TooltipPrimitive/index.tsx

```diff
@@ -20,7 +20,7 @@ export const TooltipWrapper = React.forwardRef<
       className={cx(
         "orbit-tooltip-wrapper",
         "max-w-full cursor-auto",
-        "focus:outline-none active:outline-none [&_:disabled]:pointer-events-none",
+        "focus:outline-none active:outline-none [&_:disabled]:pointer-events-none peer-checked:[&_svg]:visible",
```

Comment:
Is it a good idea to add this to Tooltip directly? Wouldn't it be better to fix it in the `Checkbox` component?

I assume this part `[&>svg]:invisible peer-checked:[&>svg]:visible` on the `"orbit-checkbox-icon-container"` (line 80) element doesn't work because because element with `peer` class (the input) is not a sibling of it but sibling of its parent.

Maybe we could use the `checked` value and set it on `"orbit-checkbox-icon-container"` like this `checked ? "[&>svg]:visible" : "[&>svg]:invisible"` instead of the current one? WDYT?

Reply:
I wanted to keep the "peer-checked" classes, but it makes sense to add a `checked` condition as well. With this approach, we can be sure that the checked icon will be visible even if the component in `tooltip` prop won't be `<Tooltip>`. 👍